mmcv/fileio/io.py

```diff
@@ -13,18 +15,30 @@
 }
 
 
-def load(file, file_format=None, **kwargs):
+def load(file, file_format=None, file_client_args=None, **kwargs):
     """Load data from json/yaml/pickle files.
 
     This method provides a unified api for loading data from serialized files.
 
+    Note:
+        In v1.3.15 and later, ``load`` supports loading data from serialized
+        files those can be storaged in different backends.
+
     Args:
         file (str or :obj:`Path` or file-like object): Filename or a file-like
             object.
```

Comment:
"file-like object"? Does it means `typing.IO` or just an object that has `read` attr

Reply:
yet, https://docs.python.org/3/glossary.html#term-file-object